### ISSUE TITLE
[Bug]: Unable to restore backup after upgrading to 2.2.14

### ISSUE THREAD

**Issue by ambarnes2** [advplyr/audiobookshelf#1485]
### Describe the issue

I created a backup while on version 2.2.13. I then upgraded to 2.2.14 and attempted to restore the backup, but nothing was restored. After clicking "Restore" the page spun for a moment then opened the Libraries page without restoring anything. I can confirm that the backup worked and saved information. I also attempted to restore to a backup that i ran a couple days ago with the same result.

![image](https://user-images.githubusercontent.com/54408675/217037084-28898075-221d-4c58-9ff9-a13670ba3756.png)

When reviewing the logs, it says nothing is being loaded

```
2023-02-06 17:01:09 INFO [DB] 0 Library Items Loaded
2023-02-06 17:01:09 INFO [DB] 0 Libraries Loaded
2023-02-06 17:01:09 INFO [DB] 0 Collections Loaded
2023-02-06 17:01:09 INFO [DB] 0 Playlists Loaded
2023-02-06 17:01:09 INFO [DB] 0 Authors Loaded
2023-02-06 17:01:09 INFO [DB] 0 Series Loaded
2023-02-06 17:01:09 INFO [DB] 1 Users Loaded
2023-02-06 17:01:09 INFO [Server] Socket vXcWVcq5KcVfojFwAAAJ disconnected from client "root" after 41048ms (Reason: transport close)
2023-02-06 17:01:09 INFO [Server] Socket Connected P45iBJ9blTjvihxJAAAL
2023-02-06 17:01:44 INFO [Server] Socket Connected kd-8fec3sqLMt_9gAAAN
2023-02-06 17:01:50 INFO [Server] Socket kd-8fec3sqLMt_9gAAAN disconnected from client "root" after 6279ms (Reason: transport close)
```

### Steps to reproduce the issue

1. Create Manual Backup under version 2.2.13
2. Recreate docker image under latest version
3. Log in with Root in 2.2.14
4. Restore recent backup
5. Nothing happens

### Audiobookshelf version

2.2.14

### How are you running audiobookshelf?

Docker

**Comment by hobesman** [comment]
This doesn't address the bug but in the meantime, the backup filess are just renamed zip files. You can simply rename it, unzip it, and replace the corresponding folders in the config folders.

**Comment by ambarnes2** [comment]
Thanks @hobesman, i did exactly that, but unfortunately, Audiobookshelf is not pulling the library information from the config folder even after replacing the content with what was in the backup directory

**Comment by advplyr** [comment]
Fixed in [v2.2.15](https://github.com/advplyr/audiobookshelf/releases/tag/v2.2.15)

I recommend creating a new backup and discarding old backups
